cremalab/crema-app-native
# Upgrade expo to SDK 40

## Description

Upgrades Expo to SDK 40 and also upgrades related packages to be compatible.

### 📋 Acceptance Criteria

> Checked off by the PR **Assignee(s)**

- [x] Starts on Android
- [x] Starts on iOS


### 👩‍🔬 Test Instructions

1. Pull down PR
2. Upgrade expo-cli with `npm i -g expo-cli`
3. Run `npm ci`
4. Run `npm start`
5. Ensure the app works on both iOS and Android

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked


---

>### Roles & Responsibilities
>
>#### 👨‍💻 Assignee
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Reviewer
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup
